src/cyw43_ll.h

```diff
@@ -163,6 +163,7 @@
 // Values used for STA and AP auth settings
 #define CYW43_WPA_AUTH_PSK (0x0004)
 #define CYW43_WPA2_AUTH_PSK (0x0080)
+#define CYW43_WPA3_AUTH_SAE_PSK (0x40000)
```

Comment:
I think these constants should be private to `cyw43_ll.c`.  Maybe we should move them there so they don't confuse users searching for auth settings?

Reply:
Yes I agree.